signbank/dictionary/models.py

```diff
@@ -822,6 +822,10 @@ class SignSeenChoices(models.TextChoices):
         default="", help_text="Optional comment the survey respondent can leave about the gloss"
     )
 
+    class Meta:
+        unique_together=(("gloss", "response_id"),)
```

Comment:
**polish**: use of `unique_together` [is discouraged](https://docs.djangoproject.com/en/4.2/ref/models/options/#unique-together)